langchain/package.json

```diff
@@ -694,6 +705,15 @@
     "@gomomento/sdk": "^1.51.1",
     "@gomomento/sdk-core": "^1.51.1",
     "@gomomento/sdk-web": "^1.51.1",
+    "@langchain/anthropic": "^0.2.6",
```

Comment:
I wouldn't upper bound these since we are very unopinionated on what you pass in - `*` is good